packages/accordion/src/AccordionItem.ts

```diff
@@ -23,12 +25,19 @@ import chevronIconStyles from '@spectrum-web-components/icon/src/spectrum-icon-c
 
 import styles from './accordion-item.css.js';
 
+const chevronClass = {
+    s: 'spectrum-UIIcon-ChevronRight75',
+    m: 'spectrum-UIIcon-ChevronRight100',
+    l: 'spectrum-UIIcon-ChevronRight200',
+    xl: 'spectrum-UIIcon-ChevronRight300',
+};
+
 /**
  * @element sp-accordion-item
  * @slot - The content of the item that is hidden when the item is not open
  * @fires sp-accordion-item-toggle - Announce that an accordion item has been toggled while allowing the event to be cancelled.
  */
-export class AccordionItem extends Focusable {
+export class AccordionItem extends SizedMixin(Focusable) {
```

Comment:
Again here, let's use the argument noDefaultSize within SizedMixin so it doesn't assume it needs to apply size m by default.